File: Made_for_LeaversApi/Made_for_LeaversApi/Entities/University.cs
/*University.cs - Daniel Syrén (20105070)*/
using System.ComponentModel.DataAnnotations;

namespace Made_for_LeaversApi.Entities
{
    public class University
    {
        [Key]
        public int Id { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }
        public string WebPage { get; set; }
    }

}

File: Made_for_Leavers/src/app/models/university.ts
/*university.ts - Daniel Syrén (20105070)*/
export class University {
    email: string|null;
    name: string;
    webPage: string;

    constructor(email: string|null, name: string, webPage: string) {
        this.email = email;
        this.name = name;
        this.webPage = webPage;
    }
}